### PR TITLE
Add gwcosmo

### PR DESCRIPTION
**Pull request by transientlunatic**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by transientlunatic** [comment]
Hi @cchapmanbird, @rachelmgray, @jcbayley can you each comment on this PR to confirm you're happy to be a maintainer? Thanks!

**Comment by jcbayley** [comment]
Also happy to be a maintainer

**Comment by transientlunatic** [comment]
@conda-forge/staged-recipes This recipe is now ready for review. Please let me know if there's anything I can do to help!

**Comment by transientlunatic** [comment]
@conda-forge/help-python This recipe is now ready for review, thanks! Note that we don't intend to support windows installation, however I'm not sure if there's a way to remove it from the Azure tests.

File: recipes/gwcosmo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{% set name = "gwcosmo" %}
+{% set version = "2.1.0a1" %}
```
Comment (ocefpaf) [inline recipes/gwcosmo/meta.yaml:2]: This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Please see the instructions there on how to publish in a dev channel, or ask upstream for a stable release.

**Comment by ocefpaf** [comment]
> Note that we don't intend to support windows installation, however I'm not sure if there's a way to remove it from the Azure tests.

For noarch packages that have missing dependencies on Windows, you don't need to do anything. We prefer to keep it noarch and this won't be installable on Windows anyway.

File: recipes/gwcosmo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    - matplotlib-base >=2.0
+    - numpy <=1.23.5,>=1.19
+    - pandas
+    - progressbar
+    - scipy
+    - seaborn
+    - tqdm
+
+channel_targets:
+  - conda-forge gwcosmo_dev
```
Comment (ocefpaf) [inline recipes/gwcosmo/meta.yaml:43]: This should be in a file named `conda_build_config.yaml`.

Comment (transientlunatic) [reply recipes/gwcosmo/meta.yaml]: Thanks! I've moved it to that file.

File: recipes/gwcosmo/meta.yaml [hunk 3]
```diff
@@ -59,3 +59,4 @@ extra:
     - CChapmanbird
     - rachelmgray
     - jcbayley
+#
```
[the remark this answers is not in the document]
Comment (transientlunatic) [reply recipes/gwcosmo/meta.yaml]: (The linter process didn't seem to notice a new maintainer commenting to confirm their agreement, but perhaps I wasn't sufficiently patient).

**Pull request merged**